addons/mail/static/src/core/common/message_model.js

```diff
@@ -459,6 +459,15 @@ export class Message extends Record {
         return this.isBodyEmpty && this.hasAttachments;
     }
 
+    get previewPrefix() {
+        if (this.isSelfAuthored) {
+            return markup(`<i class="fa fa-mail-reply me-1 opacity-75"/>You: `);
```

Comment:
This needs to be translatable. The "you" at minimum, but ideally the whole "sentence" as a single block, which would be an improvement compared to previous code.